code/game/machinery/doors/airlock_types.dm

```diff
@@ -5,20 +5,25 @@
 /obj/machinery/door/airlock/command
 	icon = 'icons/obj/doors/airlocks/station/command.dmi'
 	assemblytype = /obj/structure/door_assembly/door_assembly_com
+	wiretypepath = /datum/wires/airlock/command
 	normal_integrity = 450
+	security_level = 3
```

Comment:
is this plasteel(cutter, weld, pry, weld, pry), or just iron?

Reply:
If it's plasteel, that's literally vault door level. Reduce it to iron

Reply:
plasteel, i think iron is security 1

Reply:
Like i said, reduce it to iron

Reply:
i didnt see the comment update im sorry.
sure ill push that